app/screens/server/index.tsx

```diff
@@ -403,14 +411,21 @@ const Server = ({
                         disableServerUrl={disableServerUrl}
                         handleConnect={handleConnect}
                         handleDisplayNameTextChanged={handleDisplayNameTextChanged}
+                        handlePreauthSecretTextChanged={handlePreauthSecretTextChanged}
                         handleUrlTextChanged={handleUrlTextChanged}
                         keyboardAwareRef={keyboardAwareRef}
+                        preauthSecret={preauthSecret}
                         theme={theme}
                         url={url}
                         urlError={urlError}
                     />
                 </KeyboardAwareScrollView>
-                <AppVersion textStyle={styles.appInfo}/>
+                <View style={{height: 60, justifyContent: 'center', alignItems: 'flex-start', paddingLeft: 20}}>
```

Comment:
Move this style inside the get styleSheet function

Reply:
Changed: Changed: https://github.com/mattermost/mattermost-mobile/pull/9082/commits/bbb539790b5643e04ee486829f25a9104f2c316e